site/docs/pages/index.mdx

```diff
@@ -528,7 +528,14 @@ import { FrameMetadata } from '@coinbase/onchainkit/frame';
 
 <section className="css-alternate-container w-full flex items-center justify-center">
   <aside className="flex flex-col pb-10 md:flex-row md:px-2 w-[940px]">
-    <img alt="OnchainKit figma" src="/assets/onchain-figma.png" width="924" loading="lazy" />
+    <iframe 
+      style={{border: '1px solid rgba(0, 0, 0, 0.1)'}}
+      width="924"
+      height="520"
```

Comment:
base this on screensize?

Reply:
It actually resizes very well already

Reply:
mobile:
<img width="146" alt="Screenshot 2024-09-25 at 2 12 41 PM" src="https://github.com/user-attachments/assets/527cab42-0621-4627-95a9-031da0edad0f">

Ipad:
<img width="564" alt="Screenshot 2024-09-25 at 2 12 59 PM" src="https://github.com/user-attachments/assets/a5243bc3-fdc0-486a-9685-919eaa901bf2">